apps/query-server/pyproject.toml

```diff
@@ -10,6 +10,7 @@ python = ">=3.9,<3.13"
 sycamore-ai = { path = "../../lib/sycamore", develop = true, extras = ["opensearch", "local-inference"] }
 fastapi = { version = "^0.115.0", extras = ["standard"]}
 pydantic = "^2.9.2"
+sse-starlette = "^2.1.3"
```

Comment:
Make sure to re-lock in the root as well.

Reply:
Done!